regulations/static/regulations/css/scss/partials/_reg_text.scss

```diff
@@ -20,14 +20,17 @@
 }
 
 .reg-section {
-    margin: 1em 0;
+    margin: 1em auto;
     line-height: 1.7em;
     max-width: 75ch;
 
     // Temporary: Workaround because we're rendering multiple H1s in the subpart view.
     &:last-of-type {
     	margin-bottom: 3em;
     }
+    :target {
+	background: $teritary_background_color;
```

Comment:
@inkandsound I got carried away and put this in. Does this make sense for highlighting the targeted paragraph?

Reply:
@whytheplatypus Makes sense. I think it would be a little nicer with 'padding: .5em 0;' as well. You may need to write this as '&:target' for the SCSS to pick it up properly.